doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,178 @@
+.. _ug_nrf54h20_flpr:
+
+Working with the FLPR core
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *fast lightweight peripheral processor* (FLPR).
+The FLPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API:
+
+* UARTE120
+
+##List to be completed
+
+.. _vpr_flpr_nrf54h20_initiating:
+
+Using Zephyr multithreaded mode on FLPR
+***************************************
+
+The FLPR core can operate as a general-purpose core, running under the full Zephyr kernel.
+To enable the FLPR core, always include an overlay in the application core build that configures and builds the FLPR target.
```

Comment:
That is incorrect. The overlay only enables a driver on application core side that starts the FLPR (and copies code for non-xip variant).

Reply:
Shantha asked for this edit @shanthanordic @masz-nordic  Could you please clarify this?

Reply:
reverted